packages/ui/src/components/Input.vue

```diff
@@ -103,7 +103,7 @@ watchEffect(() => {
       ref="inputRef" v-model="value"
       class="$ui-base w-full bg-transparent color-inherit outline-none placeholder-color-gray-500 dark:placeholder-gray-300" :type="props.password ? 'password' : 'text'"
       :placeholder="placeholder" :disabled="disabled || readonly"
-      @blur="focused = false"
+      @blur="handleBlur"
```

Comment:
Seems we haven't the `handleBlur` function in the component.

Reply:
Oh. It is a mis-commit, I'll revert that.